spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/PropertySourceUtils.java

```diff
@@ -98,13 +102,14 @@ else if (properties.containsKey(SPRING_PROFILES)) {
 	}
 
 	/**
-	 * Throws IllegalStateException.
```

Comment:
- slightly change the documentation of the method
 - rename `u, v` to `left, right`
 - drop `String.format`, use `+` instead. for a single concat `String::format` is an overkill (plus it is a lot more expensive)